src/player.cpp

```diff
@@ -4,41 +4,45 @@
 
 using namespace BitBorn;
 
-fCoord25D Player::stageMovement(action stagedAction, float fElapsedTime) {
+fCoord25D Player::stageMovement(Action stagedAction, float fElapsedTime) {
     struct fCoord25D stagedPosition = Player::position;
     switch (stagedAction) {
-        case MOVE_FORWARD:
-            stagedPosition.x += sinf(stagedPosition.a) * fSpeed * fElapsedTime;
-            stagedPosition.y += cosf(stagedPosition.a) * fSpeed * fElapsedTime;
-            break;
-        case MOVE_BACK:
-            stagedPosition.x -= sinf(stagedPosition.a) * fSpeed * fElapsedTime;
-            stagedPosition.y -= cosf(stagedPosition.a) * fSpeed * fElapsedTime;
-            break;
-        case MOVE_LEFT:
-            // Left movement
-            stagedPosition.x += sinf(stagedPosition.a - (0.5 * 3.14159)) * fSpeed * fElapsedTime;
-            stagedPosition.y += cosf(stagedPosition.a - (0.5 * 3.14159)) * fSpeed * fElapsedTime;
-            break;
-        case MOVE_RIGHT:
-            // Right movement
-            stagedPosition.x += sinf(stagedPosition.a + (0.5 * 3.14159)) * fSpeed * fElapsedTime;
-            stagedPosition.y += cosf(stagedPosition.a + (0.5 * 3.14159)) * fSpeed * fElapsedTime;
-            break;
+    case MOVE_FORWARD:
+        stagedPosition.x += sinf(stagedPosition.a) * fSpeed * fElapsedTime;
+        stagedPosition.y += cosf(stagedPosition.a) * fSpeed * fElapsedTime;
+        break;
+    case MOVE_BACK:
+        stagedPosition.x -= sinf(stagedPosition.a) * fSpeed * fElapsedTime;
+        stagedPosition.y -= cosf(stagedPosition.a) * fSpeed * fElapsedTime;
+        break;
+    case MOVE_LEFT:
+        // Left movement
+        stagedPosition.x += sinf(stagedPosition.a - (0.5 * 3.14159)) * fSpeed * fElapsedTime;
+        stagedPosition.y += cosf(stagedPosition.a - (0.5 * 3.14159)) * fSpeed * fElapsedTime;
+        break;
+    case MOVE_RIGHT:
+        // Right movement
+        stagedPosition.x += sinf(stagedPosition.a + (0.5 * 3.14159)) * fSpeed * fElapsedTime;
+        stagedPosition.y += cosf(stagedPosition.a + (0.5 * 3.14159)) * fSpeed * fElapsedTime;
+        break;
+    default:
+        break;
```

Comment:
Okay, personal peeve of mine: I think switch statement cases should be indented, and then the contents of those cases should be indented as well, as if the `case` and `break` were curly braces. I'll leave it up to you, though.

Reply:
I can understand `case` being indented relative to `switch`, but `break` is code, while `case` is a label. It makes sense for those to be indented differently. Also, not all `case` clauses need to end in a break, there are some situations where falling through to the next clause makes sense, so they aren't really like curly braces.
If you want me to change the formatting settings to indent `case` relative to `switch`, I can do that. But it does feel like an unnecessary extra level of indentation to me.